lib/crawlers/activeAccounts.js

```diff
@@ -55,6 +54,10 @@ module.exports = {
         } 
       }
     }
-    setTimeout(module.exports.activeAccounts(api, pool, config), POLLING_TIME);
+
+    setTimeout(
+      () => module.exports.activeAccounts(api, pool, config),
+      config.pollingTime,
+    );
```

Comment:
I think this fix/improvement should go to a different PR. Otherwise, it is too many changes in one and while we discuss the rest we cannot merge this simple thing

Reply:
Done

Reply:
Needed for the pr scope I think

Reply:
The `config.pollingTime` yes indeed. It is fine @Bigomby . You can leave this here

Reply:
Too late :rofl:

Reply:
nooooo, please restore :-)

Reply:
And setTimeout sintax is cleaner!

Reply:
Restored